UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -262,11 +263,11 @@ public void OnShuttleArrival()
 
 		public void ProcessCargo(GameObject obj, HashSet<GameObject> alreadySold)
 		{
-			if (obj.TryGetComponent<PlayerScript>(out var playerScript))
+			if (obj.TryGetComponent<IGib>(out var iGib))
 			{
 				// No one must survive to tell the secrets of Central Command's cargo handling techniques.
 				Chat.AddExamineMsg(obj, "<color=red> You feel a strong force of energy run through your body before everything goes to black in the blink of the eye. </color>");
-				playerScript.playerHealth.Gib();
```

Comment:
Now I'm curious - I don't suppose the cargo shuttle tries to sell the AI eye?

Reply:
huh, thats a good point